cli/cli.go

```diff
@@ -91,6 +91,8 @@ func init() {
 	fs.IntVar(&defaults.App.PingInterval, "ping_interval", 3, "")
 	fs.IntVar(&defaults.App.StatsRefreshInterval, "stats_refresh_interval", 5, "")
 
+	fs.StringVar(&defaults.App.PingTimestampFormat, "ping_timestamp_format", "s", "")
```

Comment:
Let's call it `ping_timestamp_precision`.

Reply:
@palkan I've made this change.